setup.py

```diff
@@ -75,7 +75,7 @@ def get_git_sha():
         "colorama",
         "contextlib2",
         "croniter>=0.3.28",
-        "cryptography>=2.4.2",
+        "cryptography>=3.2.1",
```

Comment:
Could potentially add `, <4`.

Reply:
Since we're pinned under `requirements/` I don't think it's necessary. I'd pin if we'd know for sure that there's a backward-incompatible that is actually affecting us. Many times despite semver saying we may be affected, we're actually not.